alertlogic-cli

```diff
@@ -105,12 +104,12 @@ def main():
 
     if not import_dependencies():
         sys.exit(2)
-
-    context = parse_args()
+    services = alertlogic.dynapi.Services()
+    context = parse_args(services)
     args = context['args']
     try:
         init_logging(args)
-        result = run_command(args, context['commands'])
+        result = run_command(args, context['commands'], services)
```

Comment:
if you are passing `services` to run_command() might as well do all the logic there so you don't need to pass both arguments, given that `commands` are a subproduct of `services`